experimental_official_dockerfiles/master/packages.devel.txt

```diff
@@ -0,0 +1,53 @@
+# All required CUDA packages
```

Comment:
@perfinion Any ideas on how to decide if there are packages here we could remove?